packages/TraceDebugger.package/TDBMemory.class/instance/explorerContents.st

```diff
@@ -0,0 +1,18 @@
+as yet unclassified
+explorerContents
```

Comment:
A disadvantage of the new display is that it suggests the existence of dictionaries that are only used as auxiliary objects for grouping. Shall we really merge this?